solidity/random-beacon/hardhat.config.ts

```diff
@@ -0,0 +1,54 @@
+import { HardhatUserConfig } from "hardhat/config"
+
```

Comment:
Do we want to add [`@typechain/hardhat`](https://hardhat.org/guides/typescript.html#type-safe-smart-contract-interactions) or is it already included in some library we use?

Reply:
I'll add formatters/analyzers/linters in this PR including @typychain/hardhat